app/images/icons/arrow-2-up-right.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M6.4375 5C6.4375 4.44772 6.88522 4 7.4375 4H18C18.5523 4 19 4.44772 19 5V15.5625C19 16.1148 18.5523 16.5625 18 16.5625C17.4477 16.5625 17 16.1148 17 15.5625V7.41421L5.70711 18.7071C5.31658 19.0976 4.68342 19.0976 4.29289 18.7071C3.90237 18.3166 3.90237 17.6834 4.29289 17.2929L15.5858 6H7.4375C6.88522 6 6.4375 5.55228 6.4375 5Z" fill="#24272A"/>
+</svg>
```

Comment:
This icon doesn't look like it has been optimized? Here is a video on how to do that https://www.notion.so/Export-icons-as-optimized-svgs-for-code-daf05f66cf2e44fc8306d48a3cbbb930?pvs=4

Reply:
Good shout. Optimised it here https://github.com/MetaMask/metamask-extension/pull/18411/commits/de88b833aa1b439f638825f9e33036267adca7ba

Reply:
wow this is super useful i didn't know you can combine the path, thanks for the tutorial @georgewrmarshall

Reply:
Nice work!